ivy/functional/backends/numpy/elementwise.py

```diff
@@ -287,7 +287,7 @@ def floor_divide(
     out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
     x1, x2 = ivy.promote_types_of_inputs(x1, x2)
-    return np.floor(np.divide(x1, x2)).astype(x1.dtype)
```

Comment:
did this have issue when testing for the frontend? if I'm not mistaken, previously this was made like this due to the array api tests

Reply:
yeah, there were assertion errors.

Reply:
ahh I see, would you mind checking with Simone about this change, whether it'll affect the array api test? She is the main person for the array api fixing task, so just in case it'll affect her part, thanks!

Reply:
Okay, I'll do that.

Reply:
thanks!